src/PKSim.Presentation/DTO/Compounds/MolWeightParameterDTO.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Validation;
+using PKSim.Assets;
+using PKSim.Presentation.DTO.Parameters;
+
+namespace PKSim.Presentation.DTO.Compounds
+{
+   public class MolWeightParameterDTO : ParameterDTO
+   {
+      private readonly IParameter _effectiveMolWeightParameter;
+
+      public MolWeightParameterDTO(IParameter molWeightParameter, IParameter effectiveMolWeightParameter) : base(molWeightParameter)
+      {
+         _effectiveMolWeightParameter = effectiveMolWeightParameter;
+         Rules.AddRange(AllRules.All());
+      }
+
+      private static class AllRules
+      {
+         private static IBusinessRule effectiveMolWeightPositive { get; } = CreateRule.For<MolWeightParameterDTO>()
+            .Property(item => item.Value)
+            .WithRule((dto, valueInDisplayUnit) =>
+            {
+               var oldValue = dto.Parameter.Value;
+               try
+               {
+                  dto.Parameter.Value = dto.Parameter.ConvertToBaseUnit(valueInDisplayUnit);
+                  return dto._effectiveMolWeightParameter.Value >= 0;
```

Comment:
For the mol weight parameter, we define the value as **>= 15 g/mol**:

<img width="700" src="https://github.com/user-attachments/assets/cf74bf07-c8b0-437d-ba64-c1722cd8e40e" />

I think for the effective mol weight, the lower bound should be the same (@PavelBal Agree?). 

E.g. you could pass the lower bound of the "Molecular Weight" parameter as an additional argument to the dto:
`
public MolWeightParameterDTO(IParameter molWeightParameter, IParameter effectiveMolWeightParameter, double molWeightLowerBound) : base(molWeightParameter)` and then check for

`return dto._effectiveMolWeightParameter.Value >= molWeightLowerBound;`

Reply:
Agree.

Reply:
Yes, we can use the built-in min for this validation